stock_picking_analytic/models/stock.py

```diff
@@ -9,9 +9,21 @@ class StockPicking(models.Model):
 
     analytic_account_id = fields.Many2one(
         comodel_name="account.analytic.account",
-        string="Analytic account",
+        string="Analytic Account",
         states={"done": [("readonly", True)], "cancel": [("readonly", True)]},
     )
+    show_analytic_account = fields.Boolean(
+        string="Show Analytic Account",
+        compute="_compute_show_analytic_account",
+    )
+
+    @api.depends("picking_type_id", "picking_type_id.code")
+    def _compute_show_analytic_account(self):
```

Comment:
And... "No está asociado a un pedido de compra o de venta". Solo se puede indicar en albaranes hechos manualmente
Y no estoy segura de que no tenga que verse en internos.